delivery/docker-compose.yml

```diff
@@ -1,23 +0,0 @@
----
-version: '3.4'
-services:
-  db:
-    container_name: db
-    image: postgres
-    ports:
-      - 5432:5432
-    environment:
-      - POSTGRES_USER=kms
-      - POSTGRES_DB=kms
-      - POSTGRES_PASSWORD=kms
-      - PGDATA=/tmp/postgres2
-  kms:
-    container_name: kms
-    image: kms
-    environment:
-      - KMS_POSTGRES_URL=postgres://kms:kms@db/kms
-      - KMS_HOSTNAME=0.0.0.0
-    ports:
-      - 9998:9998
-    depends_on:
-      - db
```

Comment:
maybe we should move this part into the `docker-compose.yml` file at root of the project, what do you think?

Reply:
well if you don't mind I would rather keep the KMS service outside (the docker-compose has only the KMS external services)